generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_auth-expired.interceptor.ts

```diff
@@ -43,7 +43,6 @@ export class AuthExpiredInterceptor extends HttpInterceptor {
 <%_ } if (authenticationType === 'session') { _%>
     constructor(private injector: Injector,
         private stateStorageService: StateStorageService) {
-
```

Comment:
if this was lint fail how come the tests were passing it should have failed as I made lint not to force run